rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -1,41 +1,109 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:orientation="horizontal"
-        android:layout_margin="4dp"
-        android:padding="4dp">
+        android:paddingTop="4dp"
+        android:paddingBottom="4dp">
 
     <View
+            android:id="@+id/attachment_strip"
             android:layout_width="3dp"
             android:layout_height="match_parent"
             android:layout_marginRight="5dp"
             android:background="@color/inline_attachment_quote_line" />
 
     <LinearLayout
+            android:id="@+id/attachment_content"
             android:layout_width="0px"
             android:layout_height="wrap_content"
             android:layout_weight="1"
             android:orientation="vertical">
 
+        <LinearLayout
+                android:id="@+id/author_box"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center_vertical"
+                android:orientation="horizontal"
+                android:layout_marginBottom="8dp">
+
+            <ImageView
+                    android:id="@+id/author_icon"
+                    android:layout_width="16dp"
+                    android:layout_height="16dp"
+                    tools:src="@drawable/circle_black" />
+
+            <android.support.v4.widget.Space
+                    android:layout_width="8dp"
+                    android:layout_height="8dp" />
+
+            <TextView
+                    android:id="@+id/author_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="@style/TextAppearance.RocketChat.MessageAttachment.Title.Link"
```

Comment:
It would be nice if `textAppearance` is set to `@style/TextAppearance.RocketChat.MessageAttachment.Title` and `android:textColor="?android:attr/textColorLink"` manually.